pkg/controller/types/types.go

```diff
@@ -42,8 +42,8 @@ var (
 	// AppLabelKey contains the application label key
 	AppLabelKey string
 
-	// DefaultChaosType contains the type of chaos - infra/app
-	DefaultChaosType = "app"
+	// DefaultAnnotationCheck contains the annotation check of chaos - true/false
```

Comment:
// DefaultAnnotationCheck contains the default value (true) of the annotationCheck